deployment/docker-compose.yml

```diff
@@ -22,12 +22,14 @@ services:
   postgres:
     image: postgres
     container_name: postgres
+    env_file:
+      - .env
```

Comment:
I gave it another thought, because I think we want `docker-compose up` to just work without any manual configuration. Copying `.env.example` into `.env` is easy, but still some manual configuration.

Ideally, it should be necessary to create `.env` file only if you want to customize configuration for your needs. I was thinking about using `.env.local` file for customization, but then we need to add `.env.local` to `env-file` and it won't work for people who don't have `.env.local` file. Then I noticed that `docker-compose` has `--env-file` option. So I think that people who want to customize something can create `.env.local` using `.env.example` as a template and pass `--env-file .env.local`. Then we can rename `.env.example` to just `.env`, keep it in the repo and gitignore `.env.local` instead. What do you think?

Reply:
Hm.. I have to check that passing `--env-file` overrides `env_file` in compose file. If it works we can simplify running postgres as you propose

Reply:
yes, it's working as expected